modules/navier_stokes/src/userobjects/RhieChowMassFlux.C

```diff
@@ -247,6 +240,21 @@ RhieChowMassFlux::getVolumetricFaceFlux(const FaceInfo & fi) const
   return libmesh_map_find(_face_mass_flux, fi.id()) / face_rho;
 }
 
+Real
+RhieChowMassFlux::getVolumetricFaceFlux(const Moose::FV::InterpMethod m,
+                                        const FaceInfo & fi,
+                                        const Moose::StateArg & time,
+                                        const THREAD_ID /*tid*/,
+                                        bool /*subtract_mesh_velocity*/) const
```

Comment:
mooseAssert that subtract_mesh_velocity is not used?

Reply:
Great catch!